circuits/cpp/barretenberg/cpp/src/barretenberg/serialize/msgpack_impl/name_value_pair_macro.hpp

```diff
@@ -5,11 +5,11 @@
  * used in msgpack serialization. */
```

Comment:
After telling Jan to keep this as is of course I had to bump this. We have structs with lots of members : D